### PR TITLE
Prepare for the 0.10.0 release.

### PR DESCRIPTION
**Pull request by Harkishen-Singh**
## Description

_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue._

## Merge requirements

Please take into account the following non-code changes that you may need to make with your PR:

- [ ] CHANGELOG entry for user-facing changes
- [ ] Updated the relevant documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,22 +14,24 @@ We use the following categories for changes:
 
 ## [Unreleased]
 
+## [0.10.0] - 2022-02-17
+
 ### Added
 - Add Prometheus metrics support for Tracing [#1102]
```
Comment (niksajakovljevic) [inline CHANGELOG.md:20]: You've missed https://github.com/timescale/promscale/pull/1152

Comment (Harkishen-Singh) [reply CHANGELOG.md:20]: That is not required as `Add Prometheus metrics support for Tracing` covers it.

Comment (niksajakovljevic) [reply CHANGELOG.md:20]: Sure but then we should reference the PR at least :)

**Review by paulfantom — APPROVED** [review]
:shipit:

**Review by antekresic — APPROVED** [review]
:tada:

**Pull request merged**